### PR TITLE
Use a script which runs cover tests on all packages

### LINKED ISSUE #83: Use script to gather all package test coverage profiles
Instead of having to add each new package and subpackage to our .travis.yml file manually to collect the test coverage profiles, we should use a script like https://github.com/dedis/cosi/blob/master/coveralls.sh to handle this automatically.

### PR DESCRIPTION
**Pull request by Liamsi**
and uploads to the results to coveralls.io (after merging the cover files)

resolves #83

### CODE REVIEW

**Comment by c633** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -8,29 +8,14 @@ go:
...
-  - go test -coverprofile=extra25519.coverprofile ./crypto/ed25519/extra25519
-  - go test -coverprofile=sign.coverprofile ./crypto/sign
-  - go test -coverprofile=vrf.coverprofile ./crypto/vrf
-  - go test -coverprofile=merkletree.coverprofile ./merkletree
-  - go test -coverprofile=utils.coverprofile ./utils
-  - go test -coverprofile=protocol.coverprofile ./protocol
-  - go test -coverprofile=keyserver.coverprofile ./keyserver
-  - go test -coverprofile=bots.coverprofile ./bots
-
-after_success:
```
Comment (Liamsi) [inline .travis.yml]: Good point. I'll change that.

Comment (Liamsi) [inline .travis.yml]: Done in https://github.com/coniks-sys/coniks-go/pull/85/commits/1ca5702c0e21611939af6cb1f0c400e149e318a4

File: coveralls.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Simplified version of: https://github.com/h12w/gosweep/blob/master/gosweep.sh
+# by Wáng Hǎiliàng
```
Comment (Liamsi) [inline coveralls.sh]: There is actually no license: https://github.com/h12w/gosweep/

**Comment by masomel** [comment]
LGTM

**Pull request merged**